src/main/resources/anniv5Resources/localization/eng/serpentinepack/UIstrings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "${ModID}:SerpentinePack": {
+    "TEXT": [
+      "Serpentine Disciple",
+      "A disciple of the reptomancer. Uses Venemous and Cunning stances which focus on applying debuffs.",
+      "Dayvig",
+      "Pack Art - Dayvig"
```

Comment:
Credits were thought of for people who took stuff from other mods/games/etc
It's kinda assumed you did everything in this pack otherwise already. Feel free to let it stay, just a bit redundant.